src/convertToObject.js

```diff
@@ -6,7 +6,69 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.length) {
+    return {};
+  }
+
+  const rawLines = sourceString.split(';');
+
+  // filter lines without letters
+  const noEmptyLines = rawLines
+    .map((line) => {
+      return line.trim();
+    })
+    .filter((item) => item.length > 0);
+
+  const aggregatedLines = [];
+
+  for (const line of noEmptyLines) {
+    if (line.includes(':')) {
+      aggregatedLines.push(line);
+    } else {
+      aggregatedLines[aggregatedLines.length - 1] += line;
+    }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You can use the `reduce` method to transform `noEmptyLines` into `aggregatedLines` without the need for a `for...of` loop.